modules/ln_glcore/glcore.scm

```diff
@@ -250,15 +270,22 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 (define (glCoreTextureDraw x y w0 h0 t x1 y1 x2 y2 r . colors)
   (let ((entry (table-ref glCore:textures t #f)))
     (if entry  
-      (let ((w (flo (if (fx= (fix w0) 0) (vector-ref entry 2) w0)))
-            (h (flo (if (fx= (fix h0) 0) (vector-ref entry 3) h0))))
-      (if (null? glcore:cliplist) 
-         (apply glCore:TextureDrawUnClipped (append (list (flo x) (flo y) w h t (flo x1) (flo y1) (flo x2) (flo y2) (flo r)) 
-           (if (null? colors) '() (car colors))))
-         (apply glCore:TextureDrawClipped (append (list (flo x) (flo y) w h t (flo x1) (flo y1) (flo x2) (flo y2) (flo r)) 
-           (if (null? colors) '() (car colors)))))
-      ) (log-error "glCoreTextureDraw: unbound index " t))
-  ))
+        (let ((w (flo (if (fx= (fix w0) 0) (vector-ref entry 2) w0)))
+              (h (flo (if (fx= (fix h0) 0) (vector-ref entry 3) h0))))
+          (if (null? glcore:cliplist)
+              (if (pair? colors)
```

Comment:
Note to self - consider reformatting to 2 spaces as everywhere else before merging